MPower/app/src/main/java/org/sagebase/mpower/layout/CustomSurveyInstructionStepLayout.java

```diff
@@ -0,0 +1,65 @@
+package org.sagebase.mpower.layout;
+
+import android.annotation.TargetApi;
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.util.AttributeSet;
+import android.view.View;
+
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.step.Step;
+import org.researchstack.backbone.ui.step.layout.InstructionStepLayout;
+import org.sagebase.mpower.R;
+
+
+/**
+ * Created by rianhouston on 4/1/17.
+ */
+
+public class CustomSurveyInstructionStepLayout extends InstructionStepLayout {
```

Comment:
All these probably need a better name than "Custom...."  maybe have them be "MoodCustom..."?